modules/develop/pages/produce-data/configure-producers.adoc

```diff
@@ -1,69 +1,95 @@
+* Experiment with other component's properties, like the topic
+partition size. 
+* Explore how the producer batches messages. Increasing the
+value of `batch.size` and `linger.ms` can increase throughput by making the
+producer add more messages into one batch before sending it to the broker and
+waiting until the batches can properly fill up. This approach negatively impacts
+latency though. By contrast, if you set `linger.ms` to `0`
+and `batch.size` to `1`, you can achieve lower latency, but sacrifice throughput.
+
+=== Optimize for safety
+
+For applications where you must guarantee that there are no lost messages,
+duplicates, or service downtime, you can use higher durability `acks` settings.
+If you set `acks=all`, then the producer waits for a majority of replicas to
+acknowledge the message before it can send the next message, resulting in lower
+latency, because there is more communication required between brokers. This
+approach can guarantee higher durability because the message is replicated
+to all brokers.
+
+You can also increase durability by increasing the number of retries the broker
+can make in case messages are not delivered successfully. The trade-off
+is that duplicates may enter the system and potentially alter the
+ordering of messages.
+
+== Producer acknowledgment settings
+
+=== `acks=0`
 
 The producer doesn't wait for acknowledgments from the leader and doesn't retry
 sending messages. This increases throughput and lowers latency of the system at
 the expense of durability and data loss.
 
 This option allows a producer to immediately consider a message acknowledged when
 it is sent to the Redpanda broker. This means that a producer does not have to wait
-for any kind of response from the Redpanda broker. This is the least safe option
+for any response from the Redpanda broker. This is the least safe option
 because a leader-node crash can cause data loss if the data has not yet
 replicated to the other nodes in the replica set. However, this setting is useful
-in cases where you want to optimize for the highest throughput, and are willing
+when you want to optimize for the highest throughput and are willing
 to risk some data loss.
 
-Because of the lack of guarantees, this setting is the most network bandwidth-
-efficient. This makes it useful for use cases like IoT/sensor data collection,
+Because of the lack of guarantees, this setting is the most network bandwidth-efficient. This makes it useful for use cases like IoT/sensor data collection,
 where updates are periodic or stateless and you can afford some degree of data
 loss, but prefer gathering as much data as possible in a given time interval.
 
-==== `acks=1`
+=== `acks=1`
 
-The producer waits for an acknowledgment from the leader, but it doesn't wait
+The producer waits for acknowledgments from the leader, but it doesn't wait
```

Comment:
this change is not needed.. "The producer waits for an acknowledgment from the leader," is correct.